playbooks/deploy-machines-setup.yml

```diff
@@ -25,22 +25,16 @@
     - name: Include checking server OS compatibility
       include_tasks: tasks/check-server-os-compatibility.yml
 
-    # Configure server
-    - name: Run "apt-get update"
-      apt:
-        update_cache: yes
-        cache_valid_time: 900
-      become: True
-
     - name: Install packages
```

Comment:
We should refactor this out in a helper task that accepts a list of packages. 

then we ensure the `update_cache cache_valid_time state` values are always the same for every call

Reply:
@MSevey maybe you would like to reconsider your change request, this is pretty good improvement as is and making it a helper task could be resolved in potential follow up

Reply:
yea fine as a follow up